lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -110,6 +122,22 @@ export default class BitcoinProcessor {
         config.bitcoinWalletImportString,
         config.requestTimeoutInMilliseconds || 300,
         config.requestMaxRetries || 3);
+
+    this.lockResolver = new LockResolver(this.bitcoinClient);
+
+    this.mongoDbLockTransactionStore = new MongoDbLockTransactionStore(config.mongoDbConnectionString, config.databaseName);
+
+    const valueTimeLockTransactionFeesInBtc = config.valueTimeLockTransactionFeesAmountInBitcoins || 0.25;
+    const numberOfBlocksInOneMonth = BitcoinClient.estimatedNumberOfBlocksInOneHour * 24 * 30;
+
+    this.lockMonitor =
+      new LockMonitor(
+        this.bitcoinClient,
+        this.mongoDbLockTransactionStore,
+        config.valueTimeLockPollPeriodInSeconds || 10 * 60,
+        config.valueTimeLockAmountInBitcoins * BitcoinProcessor.satoshiPerBitcoin, // Desired lock amount in satoshis
```

Comment:
We should make sure this number comes out to be a whole number

Reply:
Will add checks inside the LockMonitor constructor.